Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/TeleOpComp.java

```diff
@@ -108,9 +98,15 @@ public void runOpMode() {
             // checks old position of arm, right when it goes over top of robot from front to back, it reduces speed
             if (gamepad2.dpad_up){
                 if (!isPressed) {
+                    if (position == -2){
+                        increase = 4;
+                    }
+                    else {
```

Comment:
These should be on the same line
```suggestion
                    } else {
```